Update post to add link to newer blog post with updated performance data (#2635)
OpenLiberty/blogs: posts/2020-10-21-memory-footprint-throughput.adoc

:url-prefix:
:url-about: /

> **NOTICE**: _This blog post has been updated with more recent performance data.  Use the following link to access the blog post containing the newer performance data._  link:{url-prefix}/blog/2022/10/06/memory-footprint-throughput-update.html[How does Open Liberty’s performance compare to other cloud-native Java runtimes]

Reviewer: ```suggestion
**NOTICE**: _This blog post is superseded by a post with more recent performance data.  Use the following link to access the blog post that uses the newer performance data._  link:{url-prefix}/blog/2022/10/06/memory-footprint-throughput-update.html[How does Open Liberty’s performance compare to other cloud-native Java runtimes]
```
